pkg/monitortests/testframework/alertanalyzer/alerts.go

```diff
@@ -364,10 +358,7 @@ func createEventIntervalsForAlerts(ctx context.Context, alerts prometheustypes.V
 		}
 
 	default:
-		ret = append(ret, monitorapi.NewInterval(monitorapi.SourceAlert, monitorapi.Error).
-			Locator(monitorapi.NewLocator().AlertFromNames("all", "", "", "", "")).
-			Message(monitorapi.NewMessage().HumanMessagef("unhandled type: %v", alerts.Type())).
-			Build(startTime, time.Now()))
+		logrus.WithField("type", alerts.Type()).Warning("unhandled prometheus alert type received in alert monitor")
```

Comment:
Not sure what this was for, it doesn't seem to happen. Switching to a log message we can hunt for in search.ci rather than injecting an interval with little to no data.